web/backend/test/plugins/metrics.test.ts

```diff
@@ -56,7 +56,6 @@ test('metrics with runtime', async (t) => {
     'nodejs_heap_space_size_used_bytes',
     'nodejs_heap_space_size_available_bytes',
     'nodejs_version_info',
-    'nodejs_gc_duration_seconds',
```

Comment:
Removed since this metric is not always available when running tests (and make this test flaky)